conans/client/build/cmake.py

```diff
@@ -45,6 +47,7 @@ def __init__(self, conanfile, generator=None, cmake_system_name=True,
         self._cmake_program = os.getenv("CONAN_CMAKE_PROGRAM") or cmake_program or "cmake"
 
         self.generator = generator or get_generator(conanfile.settings)
+        self.generator_platform = generator_platform or get_generator_platform(conanfile.settings)
```

Comment:
Just a comment: I think it makes sense to expose this as an attribute as an attribute as it can be changed before invoking the command line

Reply:
but it's already exposed the same way as `generator`, isn't it?

Reply:
Yes. I am trying to say that this is right because its value is not used as a pre-calculated definition (so changing the value after class init will not refresh the definition automatically) and it only affects the command line